Brokerages/Oanda/OandaRestApiBase.cs

```diff
@@ -143,6 +148,20 @@ protected OandaRestApiBase(OandaSymbolMapper symbolMapper, IOrderProvider orderP
             AccountId = accountId;
             Agent = agent;
             Aggregator = aggregator;
+            _subscriptionManager = new EventBasedDataQueueHandlerSubscriptionManager();
+            _subscriptionManager.SubscribeImpl += (s, t) =>
+            {
+                Log.Trace("OandaBrokerage.Subscribe(): {0}", string.Join(",", s.Select(x => x.Value)));
```

Comment:
Maybe we should we move the `Subscribe/Unsubscribe` logs into the base `DataQueueHandlerSubscriptionManager` and remove it from the different consumers, so we normalize & centralize behavior regarding it

Reply:
agree. done